source/includes/usage-examples/code-snippets/deleteMany.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri = "mongodb+srv://<username>:<password>@<cluster-address>/test?w=majority"
+
+func main() {
+	client, err := mongo.Connect(context.TODO(), options.Client().ApplyURI(uri))
+	
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		if err = client.Disconnect(context.TODO()); err != nil {
+			panic(err)
+		}
+	}()
+
+	// begin deleteMany
+	coll := client.Database("sample_mflix").Collection("movies")
+	filter := bson.D{{"runtime": bson.D{{"$gt": 800}}}}
```

Comment:
As they are just Go structs, `bson.D`s can't be created with colons in this way.
```suggestion
	filter := bson.D{{"runtime", bson.D{{"$gt", 800}}}}
```